### DISCUSSION TITLE
Welcome to python_ics Discussions!

### DISCUSSION THREAD

**Discussion by drebbe-intrepid** [intrepidcs/python_ics#190] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EFluhr** [comment]
Hey I am wondering if you have a solution for this. I am on a fresh install of 914.14. When I try to use transmit_messsages it says that ics does not have that attribute...

**Reply by drebbe-intrepid** [reply]
Can you give me the complete output?

**Reply by drebbe-intrepid** [reply]
and the output of `pip freeze`

**Reply by EFluhr** [reply]
I got it working with python_can! Thank you for your response

**Reply by EFluhr** [reply]
I got some time real quick so I'll give you it

**Reply by EFluhr** [reply]
aenum==3.1.15
argcomplete==3.1.2
can==0.0.0
click==8.1.7
colorama==0.4.6
filelock==3.12.2
future==1.0.0
importlib-metadata==6.7.0
iso8601==2.1.0
MouseInfo==0.1.3
packaging==24.0
Pillow==9.5.0
pipx==1.2.1
psutil==5.9.8
PyAutoGUI==0.9.54
pyduinocli==0.35.0
PyGetWindow==0.0.9
PyMsgBox==1.0.9
pyperclip==1.8.2
PyRect==0.2.0
PyScreeze==0.1.30
pyserial==3.5
python-can==3.3.4
python-ics==914.14
pytweening==1.2.0
pywin32==306
PyYAML==6.0.1
tk==0.1.0
typing_extensions==4.7.1
userpath==1.9.2
windows-curses==2.3.3
wrapt==1.16.0
zipp==3.15.0
![8b396eb7-8e67-47b9-8ad6-ebd0ee4659ab](https://github.com/intrepidcs/python_ics/assets/71854213/ad30c0ae-d279-4a6d-bfdd-7906577f47ea)
